src/dmd/traits.d

```diff
@@ -1076,14 +1077,12 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
 
             int dg(Dsymbol s)
             {
-                if (includeTemplates)
-                {
-                    exps.push(new DsymbolExp(Loc.initial, s, false));
-                    return 0;
-                }
                 auto fd = s.isFuncDeclaration();
-                if (!fd)
+                if (!fd) {
```

Comment:
Left curly brace on its own line, [as specified in point 3](https://github.com/dlang/dmd/blob/master/CONTRIBUTING.md#the-following-will-not-be-viewed-with-favor).